CMakeLists.txt

```diff
@@ -17,6 +17,8 @@ list(APPEND GLOBAL_CXXFLAGS ${DEFAULT_CXX_FLAGS})
 
 option(BUILD_SHARED_LIBS "Build Shared Libraries" ON)
 option(BUILD_COMMAND_LINE_TOOLS "Build Command Line Tools" ON)
+option(PEPARSE_LIBRARY_WARNINGS "Peparse Library Warnings" ON)
```

Comment:
Let's make this `OFF` by default. Also, make the string more descriptive, like `"Log pe-parse library warnings to stderr"`.